tests/test_mig_shared_safeinput.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# --- BEGIN_HEADER ---
+#
+# addheader - add license header to all code modules.
+# Copyright (C) 2009-2024  The MiG Project by the Science HPC Center at UCPH
+#
+# This file is part of MiG.
+#
+# MiG is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# MiG is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+# USA.
+#
+# --- END_HEADER ---
+#
+
```

Comment:
module docstring still missing here, but I added one during merge,

Reply:
I'm really not sure that requiring a docstring for tests maks much sense, something I tried to mention on another review.

Reply:
Sorry, I may have missed that comment in the pile of github emails :-/

While I agree to some extent at least that they are not crucial for test script as all such **should** be rather self-explanatory, it's a convention from PEP8 AFAICT.
https://peps.python.org/pep-0008/#documentation-strings

Thus, I think we should stick with it, but reusing a simple generic docstring for all the test scripts is fine with me. As in the one I added:
```
"""Unit tests for the migrid module pointed to in the filename"""
```

Reply:
Ok. Yeah, basically my issue with it is the same reason you mention - if the tests aren't self explanatory or it's not clear what\s being tested, well then there are serious problems with the tests and perhaps also the module boundaries. But the generic string is a far compromise so let's go with that.